doc/whatsnew/2/2.13/summary.rst

```diff
@@ -295,13 +295,13 @@ Other Changes
   the class itself.
 
   Closes #5408
-  RefsPyCQA/astroid#1392
+  Refspylint-dev/astroid#1392
 
 * Fixed false positive for ``unused-argument`` when a method overridden in a subclass
   does nothing with the value of a keyword-only argument.
 
   Closes #5771
-  RefsPyCQA/astroid#1382
+  Refspylint-dev/astroid#1382
```

Comment:
This looks very weird, but not for this PR. (Any other for that matter)

Reply:
https://github.com/pylint-dev/pylint/pull/8519 it was a sed away, so I did it.